### Hypothesis testing

**Q1.Suppose a child psychologist claims that the average time working mothers spend talking to their children is at least 11 minutes per day. 
You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with their children. 
Assume prior research suggests the population standard deviation is 2.3 minutes.Conduct a test with a level of significance of alpha = 0.05.**

#### 1. State the hypothesis

**Null Hypothesis**

Ho = μ ≤ 11 minutes

**Alternate Hypothesis**

Ha = μ > 11 minutes

#### 2. Set the significance level

α = 0.05
minutes


#### 3. Calculate the test statistic: We'll use the z-test statistic formula

 - x̄ = 11.5 minutes (sample mean)
 - μ = 11 minutes (population mean under the null hypothesis)
 - σ = 2.3 minutes (population standard deviation)
 - n = 1000 (sample size)

Z=(x̄ - μ)/(σ/√n)

In [20]:
import math

In [9]:
Z=(11.5-11)/(2.3)*(math.sqrt(1000))
Z

6.874516652539955

#### 4. Determine the critical value: 

critical value for a z-score corresponding to an alpha of 0.05 from z-distribution table is 1.645. 

**Since our calculated z-value 6.8745 which is greater than the critical value 1.645, we reject the null hypothesis.**

**Q2. A coffee shop claims that their average wait time for customers is less than 5 minutes.
To test this claim, a sample of 40 customers is taken, and their wait times are recorded.
The sample mean wait time is found to be 4.6 minutes with a standard deviation of 0.8 minutes. 
Perform a hypothesis test at a significance level of 0.05 and determine whether there is enough evidence to support the coffee shop's claim.**

**Null Hypothesis**

Ho = μ > 5 minutes

**Alternate Hypothesis**

Ha = μ < minutes

- x̄ = 4.6 minutes (sample mean)
- μ = 5 minutes (population mean)
- σ = 0.8 minutes (sample standard deviation)
- n = 40 (sample size)
- α = 0.05 (significance level)
- Degree of freedom(n-1) = 39

The t-test works well when we don't know the population standard deviation, especially for smaller groups, usually less than 30. But it's still good to use even for bigger groups, over 30. This is because of the Central Limit Theorem, which says that as our group size gets bigger, our average values act more like they're from a normal distribution. So, no matter if our group is small or big, the t-test is a reliable way to check things.

t = (x̄ - μ) / (σ / √n)

In [19]:
t=(4.6-5)/(0.8/math.sqrt(40))
t

-3.162277660168382

Looking at the z-table, the critical value is approximately -1.645.

**Since our calculated t-value -3.162 which is less than the critical value -1.645, we reject the null hypothesis.**
